configuration/concepts/drugs_concepts.csv

```diff
@@ -422,4 +422,52 @@ e981d07a-3541-4c09-9916-cc1460757c7b,,,Polio Vaccine-Inactivated Vaccine (IPV),V
 51ef2b9a-2ac7-4903-9594-c1bdb88eb10d,,,Diphtheria Tetanus Vaccine (DT),Vaccin diphtérique tétanique (dT),Drug,N/A,,,
 59381959-d0c2-4bfb-9da5-fbf75a046827,,,Diphtheria Tetanus Pertusis Vaccine (DTP),Vaccin diphtérique tétanique coquelucheux (DTP),Drug,N/A,,,
-c1c2676e-0533-40f7-89b4-f6f071817e02,,,Pneumococcal Vaccine (PCV13),Vaccin pneumocoque (PCV13),Drug,N/A,,,
\ No newline at end of file
+c1c2676e-0533-40f7-89b4-f6f071817e02,,,Pneumococcal Vaccine (PCV13),Vaccin pneumocoque (PCV13),Drug,N/A,,,
+03f0c0f8-2afb-401d-834a-f3b0ab2a3147,,,Aciclovir,Aciclovir,Drug,N/A,,,
+c5032f5b-11b8-49f2-a183-bd3d3a95df31,,,Ready-to-Use Supplementary Foods,Aliments Suplementaires Prets a l'emploi (ASPE),Drug,N/A,,,
+92836bbe-aa15-458f-97d6-96da0a5a2345,,,Aminophyllin,Aminophylline,Drug,N/A,,,
+51aba50f-c957-494b-aae9-c8137e56d303,,,Biperidene lactate,Lactate de bipéridène,Drug,N/A,,,
+f2e3032b-aa01-4bb2-9c5f-ffb1f0434a47,,,Bupivacaine & glucose,Bupivacaïne & glucose,Drug,N/A,,,
+c731f7a1-669d-43fe-89b4-1cc4655c7156,,,Calcium & vit,Calcium & vit,Drug,N/A,,,
+ebe04ba6-768f-42e5-9933-a5449b51a210,,,Activated charcoal,Charbon activé,Drug,N/A,,,
+a67db321-80d4-41f0-be6a-24f50d11c804,,,Cisatracurium,Cisatracurium,Drug,N/A,,,
+f4392a22-8b70-44b1-ae74-78ea084b76ef,,,Clomifen citrate,Clomifen citrate,Drug,N/A,,,
+28ed2e52-c83c-4956-95f2-0b4cf1c27655,,,Cortisone & antibiotic & antimycotic,Cortisone & antibio & antimycosique,Drug,N/A,,,
+18b4fab2-68da-4ea4-9f05-4664ac6d855d,,,Dextran in NaCl,Dextrane dans NaCl,Drug,N/A,,,
+f5e9b654-5ebb-4b17-8bb4-506e0e8da1bd,,,Diclofenac co,Diclofenac co,Drug,N/A,,,
+14146168-c1af-414c-b211-679cf1952d3d,,,Enoxaparin,Énoxaparine,Drug,N/A,,,
+e198d906-4a82-42bb-bc03-38d8083ca324,,,Ephedrine,éphédrine,Drug,N/A,,,
+62313920-a030-48fa-aa97-db7b3afadaba,,,Esmolol,Esmolol,Drug,N/A,,,
+153a837f-9c6b-4011-b062-0add78571d1a,,,Etomidate,Etomidate,Drug,N/A,,,
+1a615bb7-2fc4-419c-98b9-93f05e3953b8,,,Fenofibrate,Fénofibrate,Drug,N/A,,,
+0a5315d5-9599-43ac-868a-96a4278a617a,,,Fer elementaire & Acide folique,Fer elementaire & Acide folique,Drug,N/A,,,
+56a2f553-a2f8-4a85-b66d-b6d0db670515,,,Flumazenil,Flumazénil,Drug,N/A,,,
+9643b503-02ec-4fde-aeb1-a293dc8d7c75,,,Glycopyrrolate,Glycopyrrolate,Drug,N/A,,,
+b07006a3-62c1-485a-b8ef-af0de629d04c,,,Hialorunate de Na+,Hialorunate de Na+,Drug,N/A,,,
+bd8f1042-b1bd-4bb2-8b3b-fbda065e371f,,,Hialorunate de Na+ et oxymetazoline,Hialorunate de Na+ et oxymetazoline,Drug,N/A,,,
+3673e2b5-63bd-400a-a499-87d15e50d8a7,,,Hydrocortisone hemorrhoidal,Hydrocortisone hémorroïdaire,Drug,N/A,,,
+7055f997-aef8-4c05-9b18-6e002c6add03,,,Rabies immunoglobulins,Immunoglobulines antirabiques,Drug,N/A,,,
+d4af67f0-dbda-4da6-8c88-8ab9737c286e,,,Insulin 70 & 30,Insuline 70 & 30,Drug,N/A,,,
+13c78cde-f14e-4ce9-94db-67e549dc8903,,,Lait therapeutique F100,Lait therapeutique F100,Drug,N/A,,,
+7ae974f7-c8cc-4a99-aed4-bd5b8dcd623b,,,Lait therapeutique F75,Lait therapeutique F75,Drug,N/A,,,

```

Comment:
Let us combine this to just 'Formula Milk'

Reply:
Done